src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -3510,7 +3514,7 @@ def test_deployment_group_cross_tenant(self, resource_group):
             'version': '1.1.2',
             'captured': self.create_random_name('cli_crosstenantmanagedimage', 40),
             'aux_sub': '1c638cf4-608f-4ee6-b680-c329e824c3a8',
-            'rg': self.create_random_name('cli_test_cross_tenant_rg', 40),
+            'rg': another_resource_group,
             'aux_tenant': '72f988bf-86f1-41af-91ab-2d7cd011db47'
         })
         self.cmd('group create -g {rg} --location {location} --subscription {aux_sub}',
```

Comment:
delete previous resource group create command?

Reply:
There was no commands to delete resources in this test before. I assume it depends on the deletion task under the subscription to delete those resources